frontend/pages/smileLoginRoute/index.jsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import { Route } from '@shopgate/pwa-common/components';
+import { useTheme } from '@shopgate/engage/core';
+import getConfig from '../../helpers/getConfig';
+import Header from './components/Header';
+import MemberPanel from './components/MemberPanel';
+import PointsPanel from './components/PointsPanel';
+import Footer from './components/Footer';
+import { SMILE_LOGIN_ROUTE } from '../../constants';
+import styles from './style';
+
+/**
+ * @returns {JSX}
+ */
+const smileLoginRoute = () => {
+  const { colorConfig, loginPageText, imageSrcs } = getConfig();
```

Comment:
should not happen inside this component, because otherwise it will be executed for every render

Reply:
I removed imageSrcs, I think it is still cleaner to only import loginPageText in this component.